src/components/input/bl-input.ts

```diff
@@ -24,7 +24,8 @@ export type InputType =
   | "password"
   | "number"
   | "tel"
-  | "url";
+  | "url"
+  | "search";
```

Comment:
I noticed that documentation of this list is not synced for a while. Can you update the description on line 57 in this file?

Reply:
Would you like me to delete this comment directly? "Only `text`, `number`, and `password` is supported for now."

Reply:
It would be great to see all of the valid options in the documentation. But currently, it only says `InputType`. Maybe you can check if it's possible to show options there instead of just `InputType`, otherwise we may mention options in the description.

<img width="790" alt="image" src="https://github.com/Trendyol/baklava/assets/127687/98be3517-f4ad-42ca-b7f7-3975bc0c4f97">